rasa/constants.py

```diff
@@ -33,7 +33,7 @@
 CONFIG_MANDATORY_KEYS_NLU = ["language", "pipeline"]
 CONFIG_MANDATORY_KEYS = CONFIG_MANDATORY_KEYS_CORE + CONFIG_MANDATORY_KEYS_NLU
 
-MINIMUM_COMPATIBLE_VERSION = "1.3.0a2"
+MINIMUM_COMPATIBLE_VERSION = "1.6.0a1"
```

Comment:
can you please increase the version of the librabry to `1.6.0a2` and set this to `a2` as well?

if someone was working with master they will aready have models trained with `1.6.0a1` but these won't be compatible (not really relevant for end-users, but for anyone of us developing with it)